src/arch/riscv/pagetable_walker.cc

```diff
@@ -485,8 +857,10 @@ Walker::WalkerState::stepWalk(PacketPtr &write)
         read = new Packet(request, MemCmd::ReadReq);
         read->allocate();
 
-        DPRINTF(PageTableWalker,
-                "Loading level%d PTE from %#x\n", level, nextRead);
+        if (debug_condition()) DPRINTF(PageTableWalker,
+                "[%s] Get level%d PTE from %#x\n",
+                curstage == GSTAGE ? "GSTAGE" : "1stSTAGE",
+                curlevel, nextRead);
```

Comment:
this really needs curly braces. it's quite confusing to read and could lead to bugs.